source/standalone/tutorials/04_sensors/run_frame_transformer.py

```diff
@@ -139,10 +139,10 @@ def run_simulator(sim: sim_utils.SimulationContext, scene_entities: dict):
             if count % 50 == 0:
                 # get frame names
                 frame_names = frame_transformer.data.target_frame_names
-                print(f"Displaying Frame ID {frame_index}: {frame_names[frame_index]}")
                 # increment frame index
                 frame_index += 1
                 frame_index = frame_index % len(frame_names)
+                print(f"Displaying Frame ID {frame_index}: {frame_names[frame_index]}")
```

Comment:
I moved this down as we were incrementing after the print, causing it to be misaligned